composer.json

```diff
@@ -1,16 +1,17 @@
 {
-    "name": "pantheon-systems/wp_launch_check",
-    "description": "Check against a WordPress for performance and security",
-    "type": "wp-cli-package",
-    "license": "MIT",
-    "authors": [],
-    "minimum-stability": "dev",
-    "autoload": {
-        "files": [ "php/commands/launchcheck.php" ]
-    },
-    "require": {},
-    "require-dev": {
-        "symfony/finder": "~2.3",
-        "behat/behat": "~2.5"
-    }
+  "name": "pantheon-systems/wp_launch_check",
+  "description": "Check against a WordPress for performance and security",
```

Comment:
Existing typo. Maybe "Check a WordPress site for performance and security issues"?

Reply:
Updated description to be consistent in Composer and in the WPCLI annotations.